test/integ_tests/test_all_notebooks.py

```diff
@@ -59,7 +60,9 @@ def test_all_notebooks(notebook_dir, notebook_file, mock_level):
     os.chdir(root_path)
     os.chdir(notebook_dir)
     path_to_utils, path_to_mocks = get_mock_paths(notebook_dir, notebook_file)
-    with testbook(notebook_file, timeout=600) as tb:
+    # Try to use the conda_braket kernel if installed, otherwise fall back to the default value of python3
+    kernel = 'conda_braket' if 'conda_braket' in kernelspec.find_kernel_specs().keys() else 'python3'
```

Comment:
Why do we want to do this?

Reply:
We having been defaulting to the ipython kernel. As we have our own custom env, we should be utilizing that for testing, in particular on NBIs, to avoid differences between the two. Locally, developers may not have the conda_braket kernel set up so this avoids the case where the proper reqs are installed but the environment is not built.
To note, there are no functional issues with the current approach but this works towards correctly utilizing the env we use.